### DISCUSSION TITLE
Mods on Azure Blob

### DISCUSSION THREAD

**Discussion by RonBartsch** [Romanitho/Winget-AutoUpdate#660] [Q&A] [answered]
I would like to use Mods and store them in an Azure Blob storage, however the readme is not very clear on how to configure this.
As far as I understand I need to use the -modspath combined with the -AzureBlobURL parameter, but what should I enter as values?

Keep getting the error "Couldn't reach/find/compare/copy from <URL>

**Comment by KnifMelti** [comment, marked as answer]
I have not written that part, but he who did it wrote:

For AzureBlob: This requires the parameter -AzureBlobURL to be set with an appropriate Azure Blob Storage URL including the SAS token. See -AzureBlobURL for more information.

-AzureBlobURL
Used in conjunction with the -ModsPath parameter to provide the Azure Storage Blob URL with SAS token. The SAS token must, at a minimum, have 'Read' and 'List' permissions. It is recommended to set the permisions at the container level and rotate the SAS token on a regular basis. Ensure the container reflects the same structure as found under the initial mods folder.

Looking in the code https://github.com/Romanitho/Winget-AutoUpdate/blob/main/Sources/WAU/Winget-AutoUpdate/functions/Test-ModsPath.ps1
-ModsPath=AzureBlob

**Comment by RonBartsch** [comment]
Thank you that was it indeed, now it's working with these parameters
-ModsPath AzureBlob -AzureBlobURL "URL + SAS token"

I did not need to escape special characters in the URL.
